src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -77,6 +84,7 @@ endif()
 
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/..")
 target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${OFFLINE_TRANSFORMATIONS_LIB})
+target_compile_definitions(${PROJECT_NAME} PRIVATE CI_BUILD_NUMBER=\"${CI_BUILD_NUMBER}\")
```

Comment:
can we use https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/version.cmake#L112-L129 ?

Reply:
Yes, my initial implementation used this way with include (${OpenVINO_SOURCE_DIR}/cmake/developer_package/version.cmake), however there were several reasons to change it:
- We'd need to define 'repo_root' to make 'include' work fine
- Python API usually is built on CI separately after 'build+install ie', so there will be no ${OpenVINO_SOURCE_DIR} and we'll need to have separate logic
- Finally I decided that there is no need to restrict 'experienced developers' - so CI_BUILD_NUMBER shall be taken just from environment variable. If env var is not set, behavior for empty CI_BUILD_NUMBER is same as for 'custom_<gitbranch>_<sha>". So, this logic is not needed for 'pyopenvino' version and we can use very simple check of environment variable.

Reply:
But it works for private plugins - why? I suppose better to fix generic solution..
And I see `OpenVINO_SOURCE_DIR` is defined by Developer Package https://github.com/openvinotoolkit/openvino/blob/master/cmake/templates/InferenceEngineDeveloperPackageConfig.cmake.in#L10

Reply:
Ok, I was a bit confused that 'pyopenvino' was declared as standalone project and thought that it can be built alone. But actually it is built as part of 'src/bindings/python' which has InferenceEngineDeveloperPackage included and versions script is inherited from this.
So, now just added CI_BUILD_NUMBER compile definitions

Reply:
why not `addVersionDefines`?